ApeWorX/silverback
# feat(cli)!: find bots by default paths

### What I did

In an attempt to make the dev ux simpler, I allow for the path to be an option. If the path does not exist, we set the bath to `pathlib.Path.cwd()/bots/bot.py`

We check and raise if the file and/or the directory does not exist.

We are also adding a `Dockerfile` generator to the `_cli.py` file that will search the `bots/` directory for filenames. Each filename will be generated as a `Dockerfile.<filename>`. The Dockerfile will copy the filename into the `bots/` directory as `bot.py`.

fixes: #122 
fixes: #123 
fixes: SBK-564
fixes: SBK-563

### How I did it

Made minor changes to the `run` command and adding a method to generate the dockerfiles.

### How to verify it

Without a `bots/` directory in the root of the project run:
```
silverback run --network ethereum:mainnet
```
This will raise and show a:
```
FileNotFoundError: The bots directory '/path/to/silverback/bots' does not exist. You should have a `bots/` folder in the root of your project.
```
Then:
```
mkdir bots
silverback run --network ethereum:mainnet
```
This will raise and show:
```
FileNotFoundError: The bot.py file does not exist in the bots directory.
```
You can then run:
```
touch bots/bot.py
silverback run --network ethereum:mainnet
```
And you will get an error that does not relate to files missing.
You can also run:
```
touch bots/some_file.py
silverback run bots.some_file:bot
```
And you will get an error that does not pertain to the files/directory missing. 

MORE TO BE ADDED

### Checklist
